lib/tool_shed/util/admin_util.py

```diff
@@ -685,13 +685,8 @@ def reset_user_password(self, trans, **kwd):
                 user = get_user(trans, user_id)
                 password = kwd.get('password', None)
                 confirm = kwd.get('confirm', None)
-                if len(password) < 6:
-                    message = "Use a password of at least 6 characters."
-                    status = 'error'
-                    break
-                elif password != confirm:
-                    message = "Passwords do not match."
-                    status = 'error'
+                message = validate_password(trans, password, confirm)
+                if message:
```

Comment:
I think we are now missing:
```python
                    status = 'error'
```
here.

Reply:
Thanks for this catch! Just didn't notice that at all. Will fix.

Reply:
https://github.com/galaxyproject/galaxy/pull/9851